src/User32.Desktop/User32+WNDCLASSEX.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="WNDCLASSEX"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
```

Comment:
The fields of this struct are in the wrong order, which is vital to get right.
We always set `CharSet.Unicode`